www/components/com_volunteers/views/volunteer/tmpl/default.php

```diff
@@ -23,6 +23,14 @@
 <div class="row-fluid profile">
     <div class="span3 volunteer-image">
 		<?php echo VolunteersHelper::image($this->item->image, 'large', false, $this->item->name); ?>
+
+	    <?php if ($this->item->certification): ?>
+		    <div class="volunteer-certificated">
+			    <a href="https://exam.joomla.org/directory/user/<?php echo($this->item->certification) ?>" target="_blank">
+	                <img src="https://exam.joomla.org/images/badge.png" class="img-responsive" />
```

Comment:
maybe we should also rename this thing on the server to something more usefull as in theorie there could be more than one certificat